projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,39 +4,30 @@
 
 ## 11.3-a.9 - 2022-08-23
 ### Enhancements
-- Add a new recommendation for backups [#25446]
-- Add a recommendation for Jetpack Boost [#25431]
-- Added new Payment Buttons block in order to support adding multiple Payment Button blocks inline [#25297]
-- Add Pocket Casts as oEmbed provider. [#25696]
-- JITMs added to My Jetpack [#22452]
-- Podcast Blocks: Reduce the RSS feed cache timeout for podcasts to 1 hour [#25709]
-- VPBlock: Add support to pause/resume upload [#25392]
+- Payments block: add new container block to support using multiple inline Payment Button blocks. [#25297]
+- VideoPress: add support to pause/resume video upload. [#25392]
+
+### Improved compatibility
+- Podcast Player block: set default RSS feed cache timeout for podcasts to 1 hour. [#25709]
 
 ### Bug fixes
 - Carousel: ensure that clicks on rounded images in galleries will load a custom URL when it is specified. [#25410]
-- Dashboard Widget: do not show the Stats configuration links when the feature is inactive [#25653]
-- Fix issue in the license activation notice banner that appears even after being dismissed. [#25269]
-- Jetpack: fix VideoPress pkg version [#25683]
-- Move the logic for adding Jetpack carousel data to a block render filter so we can handle reusable blocks correctly [#25441]
+- Carousel: improve the logic for adding carousel data so we can handle reusable blocks correctly. [#25441]
+- Dashboard Widget: do not show the Stats configuration links when the feature is inactive. [#25653]
```

Comment:
```suggestion
- Dashboard Widget: do not show the stats configuration links when the feature is inactive. [#25653]
```
I think, technically, that it's correct not to capitalize stats here, because we're not calling it by its proper product name (Jetpack Stats for example).

Reply:
We had line like this before so it seems consistant:
- Stats: ensure the Stats column can always be displayed, even when the post type does not support comments.

Reply:
Ok, I guess the line is a bit blurry. If we're referring to stats as the data we're collecting then we wouldn't capitalize but here.. it just seems grey. If it was like that before, seems fine to keep it.

Reply:
I think it's fine for now, but agreed that in general we can capitalize for module/widget names but not when we're just talking about stats as a noun, like `Enable the Stats widget` vs `This widget will show you stats`.